### DISCUSSION TITLE
MQTT over QUIC Documentation

### DISCUSSION THREAD

**Discussion by luxium30** [emqx/emqx#13934] [General]
I wanted to ask where i can find an official and proper documentation or specification of MQTT over QUIC.

**Comment by ieQu1** [comment]
@qzhuyan

**Comment by qzhuyan** [comment]
Doc:

https://docs.emqx.com/en/emqx/latest/mqtt-over-quic/features-mqtt-over-quic.html

A draft spec could be found here:
https://groups.oasis-open.org/higherlogic/ws/public/document?document_id=71729&wg_id=99c86e3a-593c-4448-b7c5-018dc7d3f2f6
